.rubocop.yml

```diff
@@ -33,6 +33,11 @@ Metrics/MethodLength:
 Metrics/ModuleLength:
   Max: 120
 
+# Disabled since it only applies to Rails 5. Reenable when we migrate our apps to Rails 5.
```

Comment:
Is there any reason not to move to Rails 5 now? Rails 4.2 is more-or-less end-of-life right now (major security patches only), so....